pkg/render/policyrecommendation.go

```diff
@@ -109,8 +112,8 @@ func (pr *policyRecommendationComponent) Objects() ([]client.Object, []client.Ob
 	objs := []client.Object{
 		CreateNamespace(pr.cfg.Namespace, pr.cfg.Installation.KubernetesProvider, PSSRestricted),
 		pr.serviceAccount(),
-		pr.clusterRole(),
-		pr.clusterRoleBinding(),
+		pr.clusterRole(pr.cfg.ManagedCluster),
```

Comment:
We shouldn't need to pass in the cfg as an argument - these functions are already bound to the `pr` instance so can access the cfg directly.